docs/security-v2/03-04-ingress-egress-traffic.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Ingress and Egress traffic
+type: Details
+---
+
+## Ingress
+Kyma uses the [Istio Ingress Gateway](https://istio.io/latest/docs/reference/config/networking/gateway/) in order to handle all incoming traffic, manage TLS termination or handle mTLS communication between the cluster and external services. By default a `kyma-gateway` is configured as the main point of entry, which exposes all applications using the supplied domain and certificate.
+Exposing applications is done using the [API Gateway](components/api-gateway/#overview-overview) controller. 
+
+The current setup of the main gateway can be found [here](https://github.com/kyma-project/kyma/blob/master/resources/core/charts/gateway/templates/gateway.yaml). Configuration description:
+- The gateway listens on port `80` and `443`
+- Port `80` is automatically redirected to `443`
+- The TLS connection requires at least `TLSV1_2`, meaning `TLSV1_0` and `TLSV1_1` is rejected
+- Accepted cipher suites: `ECDHE-RSA-CHACHA20-POLY1305`, `ECDHE-RSA-AES256-GCM-SHA384`, `ECDHE-RSA-AES256-SHA`, `ECDHE-RSA-AES128-GCM-SHA256`, `ECDHE-RSA-AES128-SHA`
```

Comment:
Here or in TLS Management please also mention secret location and name as it's important for setup